Dockerfile-check

```diff
@@ -0,0 +1,15 @@
+ARG TAG=latest
```

Comment:
General comments on this file:

We do something similar already in auth_proxy, take a look at [Dockerfile.checks](https://github.com/contiv/auth_proxy/blob/6340df9b07a1723e2419069ebb5c0d5be6850a2e/build/Dockerfile.checks)

In general, we want to `COPY` in the code to be checked as one of the last operations in the Dockerfile (for caching reasons).  Bindmounting + SELinux = nightmare land, and it also prevents the use of `docker-machine` in general

Reply:
COPY is at the end, though this ARG was leftover from an earlier incantation (for FROM) I'll remove

Reply:
Right, you're `COPY`ing in the script, that's fine...

What I'm saying is we need to `COPY` in all the code to be checked, e.g.,:

```Dockerfile
COPY ./ /go/src/github.com/contiv/netplugin
```

We shouldn't use a bindmount on line 95 in Makefile